server/src/streamAPI.ts

```diff
@@ -106,7 +106,7 @@ export const streamUpdateWasReceived: RequestHandler = async (req, res) => {
       console.log("recording ready for roomID: ", hook);
       const recording = hook.call_recording;
       if (recording) {
-        const ref = await writeRecordingToDB(roomID, recording.url, recording.start_time, recording.end_time);
+        const ref = await writeRecordingToDB(roomID, recording.url.replace("ohio.stream-io-cdn.com/", "storage.googleapis.com/is-this-thing-on-recordings//"), recording.start_time, recording.end_time);
```

Comment:
yes, the double-slash is intentional — I'm not sure why, but Stream uploads files within nested folders i.e.

```
is-this-thing-on-recordings
└ /
  └ stream_recordings/
```

instead of

```
is-this-thing-on-recordings
└ stream_recordings/
```